dkpro-core-io-imscwb-asl/src/test/resources/tuebadz/corpus-sample-addfeat-ref.txt

```diff
@@ -1,41 +1,41 @@
 <text id="T990507.2">
 <s>
-Seine	PPOSAT	PRON	-	Sein
-Position	NN	NOUN	-	Position
-bei	APPR	ADP	-	bei
-der	ART	DET	-	der
-Prüfgesellschaft	NN	NOUN	-	Prufgesellschaft
-mußte	VVFIN	VERB	-	musst
-er	PPER	PRON	-	er
-damals	ADV	ADV	-	damal
-niederlegen	VVPP	VERB	-	niederleg
-,	$,	PUNCT	-	,
-den	ART	DET	-	den
-AWO-Posten	NN	NOUN	-	AWO-Post
-nicht	PTKNEG	X	-	nicht
-.	$.	PUNCT	-	.
+Seine	PPOSAT	POS_PRON	-	Sein
```

Comment:
In this file, the coarse tags should not start with POS. I.e. we should use the `coarseValue` feature instead of the simple class name.